data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -70,7 +70,7 @@
   private final int isDeletedColumnPosition;
 
   private PositionDeleteIndex deleteRowPositions = null;
-  private Predicate<T> eqDeleteRows = null;
```

Comment:
I don't see a need to remove this. Can you add it back?

Reply:
ok